File: edu.wpi.first.wpilib.plugins.cpp/resources/templates/examples/PacGoat/src/Commands/LowGoal.cpp
#include "LowGoal.h"

#include "Commands/SetPivotSetpoint.h"
#include "Commands/SetCollectionSpeed.h"
#include "Commands/ExtendShooter.h"
#include "Robot.h"

LowGoal::LowGoal() {
	AddSequential(new SetPivotSetpoint(Pivot::kLowGoal));
	AddSequential(new SetCollectionSpeed(Collector::kReverse));
	AddSequential(new ExtendShooter());
}

File: edu.wpi.first.wpilib.plugins.java/resources/templates/examples/PacGoat/src/org/usfirst/frc/team190/pacgoat/commands/LowGoal.java
package $package.commands;

import edu.wpi.first.wpilibj.command.CommandGroup;

import $package.subsystems.Collector;
import $package.subsystems.Pivot;

/**
 * Spit the ball out into the low goal assuming that the robot is in front of
 * it.
 */
public class LowGoal extends CommandGroup {
	public LowGoal() {
		addSequential(new SetPivotSetpoint(Pivot.kLowGoal));
		addSequential(new SetCollectionSpeed(Collector.kReverse));
		addSequential(new ExtendShooter());
	}
}
